internal/runners/packages/list.go

```diff
@@ -147,12 +148,16 @@ func (l *List) Run(params ListRunParams, nstype model.NamespaceType) error {
 		}
 
 		resolvedVersion := ""
-		for _, a := range artifacts {
-			if a.Namespace == ns.String() && a.Name == req.Requirement {
-				resolvedVersion = *a.Version
-				break
+		(func() {
+			for _, a := range artifacts {
+				for _, i := range a.Ingredients {
+					if i.Namespace == ns.String() && i.Name == req.Requirement {
+						resolvedVersion = i.Version
+						return // break outer loop
```

Comment:
Yeah I wrote it that way the first time and again got annoyed by the syntax so I used this instead. Labels feel like hacks to me rather than first class language features.. why are they not respecting the regular indentation?! They trigger my OCD.. :p

Reply:
Fair enough. My intent was to raise awareness that we found a legit use for the keyword, not that you had to use it :)